README.md

```diff
@@ -104,8 +104,7 @@ The OpenFGA team will do its best to address all production issues with high pri
 
 See [CONTRIBUTING](https://github.com/openfga/.github/blob/main/CONTRIBUTING.md).
 
-[doc]: https://docs.openfga.dev
-[config-doc]: https://docs.openfga.dev/configuration
-[examples]: https://docs.openfga.dev/examples
-[api]: https://docs.openfga.dev/api
-[prod-checklist]: https://docs.openfga.dev/production-checklist
\ No newline at end of file
+[doc]: https://openfga.dev/docs
```

Comment:
Are these meant to be a list of name followed by its URL or links (such as line 89-92)?

Reply:
I think that this [footnote syntax](https://www.markdownguide.org/extended-syntax/#footnotes) lets us use these terms anywhere in the document, creating a sort of reusable set of links.

I didn't write this section though, I'm just updating the links, so I'm open to discussion about whether this is the best way to use them. :)

Reply:
They are markdown reference style links, yeah.

Why the change from `-` to `+` though?